cosmwasm/contracts/outpost/src/msg.rs

```diff
@@ -27,27 +26,25 @@ pub struct Wasm {
 #[cw_serde]
 pub struct Callback {
     pub contract: Addr,
-    pub msg: String,
-}
-
-#[derive(Serialize)]
-struct PrivateCallback {
-    pub contract: String,
-    pub msg: serde_cw_value::Value,
+    pub msg: crosschain_swaps::msg::SerializableJson,
 }
```

Comment:
Private Callback was just unused right?

Reply:
It was being used for serializing. It was basically the same but it derived Serialize, which meant we could convert it to json. Now SerializableJson takes care of that